src/client/datascience/dataScienceSurveyBanner.ts

```diff
@@ -122,7 +121,6 @@ export class DataScienceSurveyBanner implements IJupyterExtensionBanner, IExtens
         this.setPersistentState(BannerType.DSSurvey, DSSurveyStateKeys.ShowBanner);
```

Comment:
We could pull the DSSurvey banner as well I think. Believe that is webviews only.

Reply:
I've left that code as is, didn't want to chagne that.
I think we have a seperate issue for banners/surveys in native notebook. hence left this.